editor/src/components/editor/actions/actions.tsx

```diff
@@ -5306,6 +5333,43 @@ export const UPDATE_FNS = {
 
     return updatedEditor
   },
+  APPLY_COLLAB_FILE_UPDATE: (action: ApplyCollabFileUpdate, editor: EditorModel): EditorModel => {
+    return modifyParseSuccessAtPath(
+      action.fullPath,
+      editor,
+      (success) => {
+        return success
+      },
+      false,
+    )
+  },
```

Comment:
This whole action can go in the bin now that we're not using it. I think this was a leftover from the first implementation

Reply:
Fixed in 4a4487461.